cleanlab_studio/internal/constants.py

```diff
@@ -36,6 +36,6 @@
 TLM_SIMILARITY_MEASURES: Set[str] = {"semantic", "string"}
 TLM_REASONING_EFFORT_VALUES: Set[str] = {"none", "low", "medium", "high"}
 TLM_VALID_LOG_OPTIONS: Set[str] = {"perplexity", "explanation"}
-TLM_VALID_GET_TRUSTWORTHINESS_SCORE_KWARGS: Set[str] = {"perplexity"}
+TLM_VALID_GET_TRUSTWORTHINESS_SCORE_KWARGS: Set[str] = {"perplexity", "constrain_outputs"}
 TLM_VALID_KWARGS: Set[str] = {"constrain_outputs"}
```

Comment:
[nit] I think would be clearer to make this more explicit distinction?

```suggestion
TLM_VALID_GET_TRUSTWORTHINESS_SCORE_KWARGS: Set[str] = {"perplexity", "constrain_outputs"}
TLM_VALID_PROMPT_KWARGS: Set[str] = {"constrain_outputs"}
```